composer.json

```diff
@@ -37,7 +37,7 @@
         "phpunit/phpunit-mock-objects": "^4.0",
         "sebastian/comparator": "^2.0",
         "sebastian/diff": "^1.4.3 || ^2.0",
-        "sebastian/environment": "^3.0.2",
+        "sebastian/environment": "^3.1@dev",
```

Comment:
Could 3.1 be deployed ?
alternatively, I could use php polyfilled constant from `symfony/polyfill`.